### ISSUE TITLE
Hermite polynomials

### ISSUE THREAD

**Issue by erex** [dill/Distance2#59]
Complete numpty about this.

This is from Planet Math  http://planetmath.org/hermitepolynomials

![planmat](https://cloud.githubusercontent.com/assets/330744/7775090/f4cc936c-00a7-11e5-933d-9db7dfe81ba1.PNG)

whereas from RDistance
![hermite-rdistance](https://cloud.githubusercontent.com/assets/330744/7775075/d02dac9e-00a7-11e5-8aca-98dea5c9b650.PNG)

I used the **RDistance** version and got the type of pdf plot I expected, better conformance to data.

**Comment by dill** [comment]
In Buckland (1992a; 1992b), Steve makes reference to Stuart and Ord (1987). It's possible that a different generating function is used.

Indeed in Buckland (1992a) we see that the first five hermite polynomials are defined as:

```{fortran}
H(O)=1.0
H(1)=XS
H(2)=XS*XS-1.0
H(3)=XS**3-3.0*XS
H(4)=XS**4-6.0*XS*XS+.30
H(5)=XS**5-10.0*XS**3+15.0*XS
```

Aparrently different again!

The other goto reference here is [Abramowitz and Stegun](http://people.math.sfu.ca/~cbm/aands/page_773.htm), who describe a few ways of calculating the Hermite terms.

My guess is that we should ensure that the results match those in Distance?

I will look into this more thoroughly. Can you point me to where you found the RDistance documentation? I can only see example analyses.



## References

Buckland, S. T. (1992a). Algorithm AS 270: Maximum Likelihood Fitting of Hermite and Simple Polynomial Densities. Applied Statistics, 41(1), 241. http://doi.org/10.2307/2347650

Buckland, S. T. (1992b). Fitting Density Functions with Polynomials. Applied Statistics, 41(1), 63. http://doi.org/10.2307/2347618

Stuart, A. and Ord, J.K. (1987) Kendall's Advanced Theoryof Statistics,pp. 220-227. London:
Griffin.

**Comment by dill** [comment]
Thanks to @wrathematics for implementing the Hermite polynomial adjustments in a more numerically stable way in C!

Merged PR in 175d6057ddbbbe55610701c8b0d2a95e87627a24.
